LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -358,5 +358,21 @@ std::wstring Utility::FormatEventLineLog(_In_ std::wstring customLogFormat, _In_
         }
     }
 
+    SanitizeCustomLog(customLogFormat);
+
     return customLogFormat;
+}
+
+void Utility::SanitizeCustomLog(_Inout_ std::wstring& customLog)
+{
+    auto npos = customLog.find(L"|");
+    std::wstring substr;
+    if (npos != std::string::npos)
+        substr = customLog.substr(npos + 1);
+        substr.erase(std::remove(substr.begin(), substr.end(), ' '), substr.end());
+
+    if (!substr.empty() && CompareWStrings(substr, L"JSON"))

```

Comment:
Maybe I missed something, why are we adding "| json" to the configuration file

Reply:
@iankingori can we have a discussion on this approach maybe during our standups

Reply:
This is to specify the custom format for formats such as JSON that need sanitization.